bundles/org.openhab.binding.gardena/README.md

```diff
@@ -115,6 +115,17 @@ DateTime LastUpdate "LastUpdate [%1$td.%1$tm.%1$tY %1$tH:%1$tM]" { channel="gard
 openhab:send LastUpdate REFRESH
 ```
 
+### Server Call Rate Limitation
+
+The Gardena server imposes call rate limits to prevent malicious use of its API.
+The limits are a) not more than one call every 15 minutes, and b) not more that 3000 calls per month.
+Normally the openHAB binding does not exceed these limits.
+But from time to time the server may nevertheless consider the limits to have been exceeded.
+In which case it reports an HTTP 429 Error (Limit Exceeded).
```

Comment:
Nit: '...have been exceeded, in which case' ... it seems a bit weird to write this as two sentences?

Reply:
Done.